agent/app/agent_test.go

```diff
@@ -156,48 +154,6 @@ func TestDoStartNewStateManagerError(t *testing.T) {
 	assert.Equal(t, exitcodes.ExitTerminal, exitCode)
 }
 
-func TestDoStartTaskLimitsFail(t *testing.T) {
```

Comment:
This [test](https://github.com/aws/amazon-ecs-agent/pull/1357/files#diff-4e5ad8207820cdb7351cbdda04bfc94cR534) is same as this removed test here.